content/blog/2019/02/2019-02-08-let-s-celebrate-java-11-support.adoc

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "Let's celebrate Java 11 Support on Jenkins"
+tags:
+- core
+- developer
+- java11
+- community
+- platform-sig
+author: alecharp
+links:
+  gitter: /jenkinsci/platform-sig
+  googlegroup: jenkins-platform-sig
+  sig: platform
+---
+
+NOTE: This is a joint blog post prepared by the Java 11 Support Team: Adrien Lecharpentier, Ashton Treadway, Baptiste Mathus, Jenn Briden, Kevin Earls, María Isabel Vilacides, Mark Waite, Ramón León and Oleg Nenashev.
+
+image:/images/logos/formal/256.png[Jenkins Java, role=center, float=right]
+
+We have worked hard for this and it's now here.
+We are thrilled to announce **full support for Java 11 in Jenkins** starting from Jenkins 2.164 and LTS 2.164.1.
+This means you can now run your instance with a Java 11 JVM.
+
+Starting in June 2018, many events were organized to improve Jenkins code base and add Java 11 support.
+Beyond these events, Core/Plugins maintainers and many other contributors have worked hard to make sure they discover and solve as many issues as possible related to Java 11 support.
+
+The effort to support Java 11 led to the creation of the https://github.com/jenkinsci/jep/blob/master/jep/211/README.adoc[JEP-211: Java 10+ support in Jenkins].
+It also spurred the creation of the https://jenkins.io/sigs/platform[Platform Special Interest Group] to coordinate the Java 11 work and other platform support efforts.
+
+== Celebration
+
+We'd like to take a moment to thank everyone involved in these tasks: code contributors, issue reporters, testers, event planners and attendees and all those in the community who have generously lent their time and support to this effort.
+Thank you all!
+
+Here are some of the contributors who helped with this task (alphabetical order):
+
+Alex Earl,
+Alyssa Tong,
+Baptiste Mathus,
+Carlos Sanchez,
+Daniel Beck,
+David Aldrich,
+Denis Digtyar,
+Devin Nusbaum,
+Emeric Vernat,
+Evaristo Gutierrez,
+Gianpaolo Macario,
+Isabel Vilacides,
+James Howe,
+Jeff Pearce,
+Jeff Thompson,
+Jesse Glick,
+Jonah Graham,
+Kevin Earls,
+Ksenia Nenasheva,
+Kohsuke Kawaguchi,
+Liam Newman,
+Mandy Chung,
+Mark Waite,
+Nicolas De Loof,
+Oleg Nenashev,
+Oliver Gondža,
+Olivier Lamy,
+Olivier Vernin,
+Paul Sandoz,
+Ramón León,
+Sam Van Oort,
+Tobias Getrost,
+Tracy Miranda,
+Ulli Hafner,
+Vincent Latombe,
+Wadeck Follonier,
+
+== Guidelines
+
+Running Jenkins on Java 11 with Docker is simple.
```

Comment:
Would be nice to reference other run options so that it does not look like Docker-only